telegram/ext/commandhandler.py

```diff
@@ -129,8 +142,8 @@ def check_update(self, update):
             :obj:`bool`
 
         """
-        if (isinstance(update, Update)
-                and (update.message or update.edited_message and self.allow_edited)):
+        if (isinstance(update, Update) and
+                (update.message or update.edited_message and self.allow_edited)):
```

Comment:
Since it's deprecated - remove this code.
(and don't forget to update CHANGES.rst)

Reply:
That seems like something for another PR.

Reply:
Everything deprecated with Handlers - this is the time...